Gemfile.lock

```diff
@@ -347,33 +342,33 @@ GEM
       netrc (~> 0.8)
     rspec-core (3.8.2)
       rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.4)
+    rspec-expectations (3.8.6)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-mocks (3.8.1)
+    rspec-mocks (3.8.2)
```

Comment:
### [rspec-mocks](https://github.com/rspec/rspec-mocks)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.8.1 → 3.8.2

[_[change-log](https://github.com/rspec/rspec-mocks/blob/v3.8.2/Changelog.md), [source-code](https://github.com/rspec/rspec-mocks)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-mocks/compare/v3.8.1...v3.8.2).

These are the individual commits:
- (42962f4) [Update Aruba to version 0.14.10 (#1279)](https://github.com/rspec/rspec-mocks/commit/42962f41a36bf0adf9459fece3095465140f6008)
- (c7ae545) [Updated travis build scripts (from rspec-dev) (#1282)](https://github.com/rspec/rspec-mocks/commit/c7ae545d801fe5384ee86b27b08477b68f444d0a)
- (4584864) [Updated travis build scripts (from rspec-dev) (#1285)](https://github.com/rspec/rspec-mocks/commit/4584864404fe16cc897da98a10750755f2d29f26)
- (674381c) [Fix bug where array_including argument matcher could not be …](https://github.com/rspec/rspec-mocks/commit/674381c3d92418b7773c86c449e1834a827688f7)
- (ef647d7) [Change log for #1291](https://github.com/rspec/rspec-mocks/commit/ef647d71441e6ee8aa192994e00b4b9a42dd747c)
- (b2763ac) [Version 3.8.2](https://github.com/rspec/rspec-mocks/commit/b2763ac8f725579f9caff0d840ae639abda9347e)


</details>